### PR TITLE
Adding 2.7.1 release notes

### PR DESCRIPTION
**Pull request by horzsolt**
# Description

Update the release notes with 2.7.1

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

### CODE REVIEW

**Review by mkindahl — APPROVED** [review]

**Review by svenklemm — APPROVED** [review]

File: timescaledb/overview/release-notes/index.md [hunk 1]
```diff
@@ -71,9 +71,12 @@ exclude all but the most recent chunk were dominated by planning time.
 This new optimization enhances the planner to take the `now()` constraint into
 consideration when planning thereby reducing the work that needs to be done
 in the planner.
+
+This release also contains bug fixes and minor feature improvements since the 2.7.0 release.
+
 <!-- <highlight type="note"> This release is low priority for upgrade. We recommend that you upgrade when you can. </highlight> -->
 
-<highlight type="important"> This release is medium priority for upgrade. We recommend that you upgrade at the next available opportunity. </highlight>
+<highlight type="important"> This release is a patch release. We recommend that you upgrade at the next available opportunity. </highlight>
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:79]: These are standard phrases, so we shouldn't really be changing them. Can we add the patch release info elsewhere, and leave this with the original phrasing?

File: timescaledb/overview/release-notes/index.md [hunk 2]
```diff
@@ -110,6 +113,35 @@ for more information and links to installation instructions when upgrading from
...
+* #4367 Improved buffer management in the copy operator
+* #4375 Don't ask for orderby column if default already set
+* #4400 Use our implementation of `find_em_expr_for_rel` for PG15+
+* #4408 Fix crash during insert into distributed hypertable
+* #4411 Add `shmem_request_hook`
+* #4437 Fix segfault in subscription_exec
+* #4442 Fix perms in copy/move chunk
+* #4450 Retain hypertable ownership on `attach_data_node`
+* #4451 Repair numeric partial state on the fly
+* #4463 Fix empty bytea handlng with distributed tables
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:131]:
```suggestion
* #4463 Fix empty byte handling with distributed tables
```

Comment (horzsolt) [reply timescaledb/overview/release-notes/index.md:131]: It should remain bytea, this is the name of this type:
https://www.postgresql.org/docs/current/datatype-binary.html

File: timescaledb/overview/release-notes/index.md (hunk 1 above)
Comment (horzsolt) [reply timescaledb/overview/release-notes/index.md]: Thanks, I've updated as you suggested.

File: timescaledb/overview/release-notes/index.md [hunk 3]
```diff
@@ -110,6 +113,35 @@ for more information and links to installation instructions when upgrading from
 </highlight>
 
 ## Release notes
+## 2.7.1 (2022-07-07)
+
+**Bugfixes**
+* #4494 Handle timescaledb versions aptly in multinode
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:119]:
```suggestion
* #4494 Handle TimescaleDB versions aptly in multi-node
```

File: timescaledb/overview/release-notes/index.md [hunk 4]
```diff
@@ -110,6 +113,35 @@ for more information and links to installation instructions when upgrading from
...
+* #4493 Segfault when executing IMMUTABLE functions
+* #4482 Fix race conditions during chunk (de)compression
+* #4367 Improved buffer management in the copy operator
+* #4375 Don't ask for orderby column if default already set
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:123]:
```suggestion
* #4375 Don't ask for `orderby` column if default already set
```

File: timescaledb/overview/release-notes/index.md [hunk 5]
```diff
@@ -110,6 +113,35 @@ for more information and links to installation instructions when upgrading from
...
+* #4400 Use our implementation of `find_em_expr_for_rel` for PG15+
+* #4408 Fix crash during insert into distributed hypertable
+* #4411 Add `shmem_request_hook`
+* #4437 Fix segfault in subscription_exec
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:127]:
```suggestion
* #4437 Fix segfault in `subscription_exec`
```

File: timescaledb/overview/release-notes/index.md (hunk 2 above)
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:131]:
```suggestion
* #4463 Fix empty byte handling with distributed tables
```

File: timescaledb/overview/release-notes/index.md [hunk 6]
```diff
@@ -110,6 +113,35 @@ for more information and links to installation instructions when upgrading from
...
+* #4469 Better superuser handling for `move_chunk`
+
+**Features**
+* #4244 Function telemetry
+* #4287 Add internal api for foreign table chunk
+* #4470 Block drop chunk if chunk is in frozen state
+* #4464 Add internal api to associate a hypertable with custom jobs
+
+**Thanks**
+* @xin-hedera Finding bug in empty bytea values for distributed tables
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:141]:
```suggestion
* @xin-hedera Finding bug in empty byte values for distributed tables
```

File: timescaledb/overview/release-notes/index.md (hunk 2 above)
Comment (horzsolt) [reply timescaledb/overview/release-notes/index.md:131]: it should remain bytea

File: timescaledb/overview/release-notes/index.md (hunk 6 above)
Comment (horzsolt) [reply timescaledb/overview/release-notes/index.md:141]: it should remain bytea

File: timescaledb/overview/release-notes/index.md (hunk 2 above)
Comment (svenklemm) [reply timescaledb/overview/release-notes/index.md:131]: bytea is a postgres datatype, so the version in PR is correct
https://www.postgresql.org/docs/current/datatype-binary.html

**Pull request merged**